pages/linux/kscreen-console.md

```diff
@@ -0,0 +1,32 @@
+# kscreen-console
+
+> KScreen Console is a CLI tool to query KScreen's status.
+> More information: <https://manned.org/kscreen-console>.
+
+- Show all outputs and configuration files to attach to a bug report:
+
+`kscreen-console bug`
+
+- Show paths to KScreen configuration files:
+
+`kscreen-console config`
+
+- Show display output information:
+
+`kscreen-console outputs`
+
+- Monitor for changes:
+
+`kscreen-console monitor`
+
+- Show the current KScreen configuration in JSON:
+
+`kscreen-console json`
+
+- Display help on command-line options:
```

Comment:
```suggestion
- Display help:
```

Reply:
I like the more descriptive variant too.

Reply:
I just copied it from another command, I have no idea which. ^-^'
We're all over the place on this one, but generally, it's more verbose.

* babel - Output all available options:
* consul - Show general help:
* dvc - Display general help:
* ghdl - Display the help page:
* git - Show general help:
* grunt - List all available options:
* hakyll-init - Show help for `hakyll-init`:

Reply:
`grep -Rn 'Display help:' | wc -l` has 24 matches.

Reply:
Let's just go with `Display help` as @navarroaxel suggested, and we can discuss whether or not to include `--help` examples in a different issue (I think that's coming soon :)